mica-webapp/src/main/conf/application.yml

```diff
@@ -48,10 +48,11 @@ opalTaxonomies:
     defaultTermsSortOrder: "title"
 
 spring:
+    config.activate.on-profile: prod
+    main.allow-bean-definition-overriding: true
```

Comment:
In case of an upgrade these may not be added (keep current application.yml option). Wouldn't it be preferable to have these in the mica-core's application.yml, as default values?

Reply:
yes, didn't think of that. changed this config's location to core's application.yml.